### PR TITLE
feat: add animationMode configuration

### PR DESCRIPTION
**Pull request by ilhan007**
- Add `animationMode` to configure the degree of components animations.
Available options are: `full` (by default), `basic`, `minimal` and `none`,
where `full` means all animations run without restrictions and `none` means all animations are off.
- Switch on/off the ui5-panel toggle animation based on the `animationMode`.

### CODE REVIEW

File: packages/base/src/types/AnimationModes.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+const AnimationModes = {
```
Comment (vladitasev) [inline packages/base/src/types/AnimationModes.js]: Should be AnimationMode (not modes), similar to ButtonType.

File: docs/Configuration.md [hunk 2]
```diff
@@ -11,6 +11,7 @@ theme        | sap_fiori_3, sap_belize, sap_belize_hcb         | sap_fiori_3   |
 language     | en, de, es, etc...                              | en            | Language to be used for translatable texts
 [RTL](#rtl)          | true, false                                     | false         | When true, sets global text direction to right-to-left
 compactSize  | true, false                                     | false         | When set, enforces compact density (smaller margins/paddings)
+[animationMode](#animationMode)  | full, basic, minimal, none  | false         | Defines the animation effects of the web components
```
Comment (vladitasev) [inline docs/Configuration.md]: Set the correct one

File: packages/base/src/types/AnimationModes.js (hunk 1 above)
Comment (ilhan007) [reply packages/base/src/types/AnimationModes.js]: yes, done

File: docs/Configuration.md (hunk 2 above)
Comment (ilhan007) [reply docs/Configuration.md]: done

**Pull request merged**